packages/components/autocomplete/Autocomplete.mdx

```diff
@@ -0,0 +1,241 @@
+---
+title: 'Autocomplete'
+type: 'component'
+status: 'stable'
+slug: /components/autocomplete/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/autocomplete'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-autocomplete--basic'
+typescript: ./src/Autocomplete.tsx
+---
+
+import { Props } from '@contentful/f36-docs-utils';
+
+The autocomplete component is an input field that provides selectable options as a user types into it.
+It allows users to quickly search through and select from large collections of options.
+
+## Table of contents
+
+- [How to use Autocomplete](#how-to-use-autocomplete)
+- [Code examples](#code-examples)
+- [Content recommendations](#content-recommendations)
+- [Accessibility](#accessibility)
+
+## How to use Autocomplete
+
+The Autocomplete requires 3 props to work:
+
+- `items`: It’s an array containing the items that will be shown as selectable options when the user types something in the `TextInput`.
```

Comment:
It can be an array of anything actually

I just added an example with objects because I thought it would be a good example of how to use complex data with the Autocomplete